assets/apps/code-editor/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
+    <title>Document editor</title>
```

Comment:
Elsewhere we use Code editor. Document editor means word processor to most people.

Reply:
Yep I want to target text editing